doc/source/releases/21.01.rst

```diff
@@ -0,0 +1,683 @@
+  `Pull Request 11047`_
+* Add msf datatype
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 11084`_
+* Allow the "Apply Rules" tool to attach tags to output datasets.
+  `Pull Request 10382`_
+* Implement mypy type checking for the Galaxy Python backend.
+  `Pull Request 11009`_, `Pull Request 11037`_,
+  `Pull Request 11022`_, `Pull Request 11007`_,
+  `Pull Request 11027`_, `Pull Request 11039`_,
+  `Pull Request 11000`_, `Pull Request 10992`_,
+  `Pull Request 11013`_, `Pull Request 11019`_
+* Allow setting file download filename_pattern by client.
+  `Pull Request 11108`_
+* Add FastAPI migration of licenses endpoints
+  (thanks to `@davelopez <https://github.com/davelopez>`__).
+  `Pull Request 11056`_
+* Improvements to workflow imports from GA4GH TRS servers
+  (thanks to `@OlegZharkov <https://github.com/OlegZharkov>`__).
+  `Pull Request 10805`_, `Pull Request 10619`_
+* Major enhancements for the ``galaxy-tool-test`` script (installed as part of the PyPI
+  package ``galaxy-tool-util``).
+  `Pull Request 10620`_, `Pull Request 10779`_,
+  `Pull Request 10908`_, `Pull Request 10879`_,
+  `Pull Request 10725`_, `Pull Request 10735`_
+  `Pull Request 10767`_
+* Speedup xml replacement when loading tools.
+  `Pull Request 10804`_
+* Update askomics interactive tool to 4.1.1
+  (thanks to `@abretaud <https://github.com/abretaud>`__).
+  `Pull Request 10878`_
+* Revise tool inputs to allow nested parameters and allow ``build_for_rerun`` and other
+  (thanks to `@simonbray <https://github.com/simonbray>`__).
+  `Pull Request 11332`_, `Pull Request 10720`_
+* Allow empty file uploads
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 11079`_
+* Convert collection creator user interface components to Vue.
+  `Pull Request 10240`_
+* Disable dataset edit button after the dataset has been deleted
+  (thanks to `@simonbray <https://github.com/simonbray>`__).
+  `Pull Request 10589`_
+* Implement expression tool for picking non-null values for CWL-style merging of
+  conditional branches in native Galaxy workflows.
+  `Pull Request 10712`_
+* Add and document API for publishing and tagging workflows
+  (thanks to `@simonbray <https://github.com/simonbray>`__).
+  `Pull Request 10748`_
+* Allow relative paths when staging jobs (e.g. with planemo run)
+  (thanks to `@simonbray <https://github.com/simonbray>`__).
+  `Pull Request 10886`_
+* Various small cleanups and improvements to the workflow editor and migrations toward
+  Vue.
+  `Pull Request 10999`_, `Pull Request 11001`_, `Pull Request 11024`_,
+  `Pull Request 10769`_, `Pull Request 11036`_, `Pull Request 11042`_,
+  `Pull Request 11041`_, `Pull Request 10580`_, `Pull Request 11046`_
+* Add FastAPI as backend and start porting backend to an ASGI webapp to
+  enable modern programming features such a message queues and web sockets.
+  `Pull Request 10823`_, `Pull Request 11074`_
+*  Replace StreamBall, replace tarballs with zip
```

Comment:
```suggestion
* Replace StreamBall, replace tarballs with zip
```
I think that's what's causing it to be interpreted as a black quote.

Reply:
Thanks, fixed it.